shellbase-core/src/main/scala/com/sumologic/shellbase/ShellCommandSet.scala

```diff
@@ -153,13 +153,13 @@ class ShellCommandSet(name: String, helpText: String, aliases: List[String] = Li
   final def execute(cmdLine: CommandLine) =
     throw new IllegalAccessException("Call the other signature!")
 
-  override def argCompleter = new AggregateCompleter(commands.map(_.completer))
+  override def argCompleter = new AggregateCompleter(commands.map(_.completer).toSeq: _*)
 
   def commandVariants(command: ShellCommand): List[String] = {
     command.basicVariants.flatMap(namingConvention.nameVersions(_)).distinct
   }
 
-  def findCommand(input: String) = {
+  def findCommand(input: String): Unit = {
```

Comment:
I think that's wrong and Travis found it out.

Reply:
Yep, good catch.  I must have done this as a late edit and not re-ran UTs.  Passes with latest commit.